src/main/java/bio/terra/cli/command/groups/ListUsers.java

```diff
@@ -33,7 +33,7 @@ protected void execute() {
   /** Print this command's output in text format. */
   private static void printText(java.util.List<String> returnValue) {
     for (String user : returnValue) {
-      System.out.println(user);
+      OUT.println(user);
```

Comment:
Do the subcommands use this static variable, or should they call `getOut()`?

Reply:
As long as the sub-commands extend the `BaseCommand` class, they should just use the static variable. This variable is set in `BaseCommand` by calling `getOut()`, so there's no need for all sub-classes to repeat the call.